code.gs

```diff
@@ -1585,6 +1585,22 @@ function _ (str) {
   return i18n[settings.user.lang][str] || str;
 }
 
+/**
+ * Return whether an item exists as a value in an object.
+ *
+ * Must use "indexOf" with values rather than "in" with keys, because e.g.
+ * "null" and "undefined" can't be keys. No need for "typeof undefined"
+ * syntax for comparing "undefined" as we are not targeting browsers, let
+ * alone old ones.
+ *
+ * @param {!any} item - The item to search the values for.
+ * @param {!object} arr - The object to search in.
+ * @returns {boolean} - Whether the item exists as a value in the object.
+ */
+function isIn(item, arr) {
```

Comment:
Another simpler problem with semistandard here: a space is required between "isIn" and the parethesis.